src/QMCHamiltonians/NonLocalECPotential.h

```diff
@@ -22,54 +22,56 @@
 
 namespace qmcplusplus
 {
-
 class NonLocalECPComponent;
+
 /** @ingroup hamiltonian
  * \brief Evaluate the semi local potentials
  */
 class NonLocalECPotential : public OperatorBase, public ForceBase
 {
 public:
-  NonLocalECPotential(ParticleSet& ions,
-                      ParticleSet& els,
-                      TrialWaveFunction& psi,
-                      bool computeForces = false,
-                      bool useVP         = false);
+  /** tuple type for NLPP calculation of a pair of ion and electron
+   * ion id, electron id, electron position, ion electron distance, ion to electron displacement
+   */
+  using NLPPJob = std::tuple<int, int, PosType, RealType, PosType>;
```

Comment:
I'd favor making this a structure.  
```C++
struct NLPPJob
{
  int ion_id;
  int electron_id;
  PosType elec_pos;
  RealType ion_elec_dist;
  PosType ion_elec_disp;
}
```

Reply:
Good suggestion. I regretted after the tuple was coded. Will submit a separate PR to address this.